frontend/controller/actions/identity.js

```diff
@@ -378,7 +378,7 @@ export default (sbp('sbp/selectors/register', {
 
         const promptOptions = {
           heading: L('Login error'),
-          question: L('Do you want to log out? Error details: {err}.', { err: err.message }),
+          question: L('Do you want to log out? {br_}Error details: {err}.', { err: err.message }),
```

Comment:
- The `L` function doesn't come built-in with `{br_}`, that's provided by `LTag()`
- Please test and check to make sure that visually it looks good. E.g., do we want `{br_}` or do we want `{p_}..{_p}`?

Reply:
> The L function doesn't come built-in with {br_}, that's provided by LTag()
I'm confused by this point, as `english.strings` does contain several occurences of `{br_}` usage

Reply:
@snowteamer sure, but that's because `L` is used with `LTags` (search the codebase for examples)